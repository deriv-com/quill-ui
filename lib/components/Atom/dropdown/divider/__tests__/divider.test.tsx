import React from "react";
import { render } from "@testing-library/react";
import { Divider } from "..";

describe("Divider", () => {
    it("renders the divider with the correct class name", () => {
        const { container } = render(<Divider />);
        const divider = container.firstChild;

        expect(divider).toBeInTheDocument();
        expect(container).toMatchSnapshot();
    });
});
