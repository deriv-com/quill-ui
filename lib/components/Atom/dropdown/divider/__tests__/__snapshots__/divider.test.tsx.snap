// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Divider renders the divider with the correct class name 1`] = `
<div>
  <div
    class="quill-dropdown__divider"
  />
</div>
`;
